README.md

```diff
@@ -211,4 +213,23 @@ A list of currently active Blockchains projects. Sorted Alphabetically.
 
   #### Tutorials
    - [Complete Guide to Full Stack Solana Development by Nader Dabit](https://dev.to/edge-and-node/the-complete-guide-to-full-stack-solana-development-with-react-anchor-rust-and-phantom-3291)
- - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
\ No newline at end of file
+ - [How to Mint an NFT on Solana](https://www.quicknode.com/guides/web3-sdks/how-to-mint-an-nft-on-solana )
+ ----------
+ ## &#127916; Youtube Tutorials 
+ 
+ ## You can explore about Crypto, Blockchain,DAOs,EVM many things to learn &#11015;&#65039;
+ 1. [Dapp University](https://youtu.be/uGPC9wNTBbw)
+ 
+ 2. [Whiteboard Crypto](https://youtu.be/mSeLVtBe2xE)
```

Comment:
This link still takes me to Gala Games so I am confused, Im not sure that's the right link as I'm expecting to see Whiteboard Crypto channel?
<img width="948" alt="Screenshot 2022-08-16 at 15 49 23" src="https://user-images.githubusercontent.com/77060368/184910038-dfb90bb4-ce48-4967-8dc3-1da8b5bc154c.png">

Reply:
Ya just typo mistake #solving

Reply:
I solved this issue please check it out I think everything is fine now.